src/main/java/io/apicurio/datamodels/util/ModelTypeUtil.java

```diff
@@ -81,4 +81,22 @@ public static boolean isOpenApi2Model(Node node) {
         }
     }
 
+    public static boolean isOpenApi30Model(Node node) {
+        switch (node.root().modelType()) {
```

Comment:
Why the switch statement and not just simple comparison?

Reply:
I like the syntax **in this case** a little better, and it fits better with the pattern of the other methods in the class.